media/src/project/project-map.tsx

```diff
@@ -40,7 +41,7 @@ export const ProjectMap = () => {
                 return response.json();
             })
             .then((data) => {
-                setProjectInfo(data[0]);
+                setProjectInfo(data);
```

Comment:
At Ln. 36, I think the url setup may need a trailing slash. I'm seeing two server hits against the project api -- one without a trailing slash that returns a 301 and one with that returns a 200.